StudCity.Frontend/src/components/Group/GridCard/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export default function GridCard({ group }) {
+	return (
+		<div className="w-full h-full flex-auto">
+			<div className='h-1/2 w-full rounded-t-xl relative'>
+				<img src={group.image} alt="" className='h-full w-full rounded-t-xl' />
```

Comment:
please provide a value for 'alt' attribute in case user lost connection during website visiting

Reply:
@Vova-Romaniuk don`t miss this comment